### PR TITLE
Improve log, save origin yaml, and fix adan

### PR DESCRIPTION
**Pull request by SamitHuang**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Improve log info.
Archive original yaml config used in training to better ensure reproduction.
Fix Adan optimizer (sync mindcv)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ addict
 matplotlib
 addict
 numpy
+shutils
```
Comment (hadipash) [inline requirements.txt:8]: this is not `shutil`. `shutil` is a built-in python library, so no need to install with `pip`.

File: mindocr/data/builder.py [hunk 2]
```diff
@@ -181,7 +186,6 @@ def _check_batch_size(num_samples, ori_batch_size=32, refine=True):
         for bs in range(ori_batch_size - 1, 0, -1):
```
[the remark this answers is not in the document]
Comment (zhtmike) [reply mindocr/data/builder.py:186]: agree. The time consumption of run batch size = 1 is usually longer than the time running with two different batch sizes, the compiling time of model due to the different batch size is negligible

**Review by zhtmike — COMMENTED** [review]
The comment out print function can be removed for better readability

File: mindocr/data/builder.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (SamitHuang) [reply mindocr/data/builder.py:186]: Even when we set `drop_remainder` to False, the last batch will be padded to `batch_size`, leading to an inaccurate evaluation result.

[the remark this answers is not in the document]
Comment (zhtmike) [reply mindocr/data/builder.py:186]: I remember it is not padded, the remainder will be output with different batch size.  
```python
from mindspore.dataset import GeneratorDataset

dataset = GeneratorDataset(range(10), 'data').batch(4)
for x in dataset.create_tuple_iterator(num_epochs=1):
    print(x[0].shape)

>>>(4,)
>>>(4,)
>>>(2,)

```

**Review by zhtmike — APPROVED** [review]

**Comment by zhtmike** [comment]
btw model.eval API indeed will have undefined behaviour like padding at the end of the iteration. Should use it with care
```python
import mindspore as ms
import mindspore.nn as nn
from mindspore.dataset import GeneratorDataset

class Iter:
    def __len__(self):
        return 10
    
    def __getitem__(self, index):
        return index, index 

dataset = GeneratorDataset(Iter(), ['data', 'label'], shuffle=False).batch(4)

class PrintNet(nn.Cell):
    def construct(self, x):
        print(x)
        return x

loss = nn.MSELoss()
net = PrintNet()
model = ms.Model(net, loss, metrics={'mse'})
model.eval(valid_dataset=dataset)
>>>Tensor(shape=[4], dtype=Int64, value=[0 1 2 3])
>>>Tensor(shape=[4], dtype=Int64, value=[4 5 6 7])
>>>Tensor(shape=[4], dtype=Int64, value=[8 9 6 7])
```

File: mindocr/optim/adan.py [hunk 3]
```diff
@@ -73,67 +54,55 @@ def _update_run_op(
...
+def _check_param_value(beta1, beta2, eps, prim_name):
     """Check the type of inputs."""
-    assert isinstance(beta1, float), f"For '{prim_name}', the type of 'beta1' must be 'float', but got type '{type(beta1).__name__}'."
-    assert isinstance(beta2, float), f"For '{prim_name}', the type of 'beta2' must be 'float', but got type '{type(beta2).__name__}'."
-    assert isinstance(eps, float), f"For '{prim_name}', the type of 'eps' must be 'float', but got type '{type(eps).__name__}'."
-    assert 0.0 < beta1 < 1.0, f"For '{prim_name}', the range of 'beta1' must be (0.0, 1.0), but got {beta1}."
-    assert 0.0 < beta2 < 1.0, f"For '{prim_name}', the range of 'beta2' must be (0.0, 1.0), but got {beta2}."
-    assert eps > 0, f"For '{prim_name}', the 'eps' must be positive, but got {eps}."
-    assert isinstance(use_locking, bool), f"For '{prim_name}', the type of 'use_locking' must be 'bool', but got type '{type(use_locking).__name__}'."
+    assert isinstance(beta1, float) and 0 <= beta1 <= 1.0, f"For {prim_name}, beta1 should between 0 and 1"
```
Comment (HaoyangLee) [inline mindocr/optim/adan.py:103]: Update the docstring of function `_update_run_op()` in line 44 and 45. `(0.0, 1.0)` -> `[0.0, 1.0]`

**Review by HaoyangLee — APPROVED** [review]
